common/io.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#include <common/io.hpp>
+
+namespace mender::common::io {
+
+StdError Copy(Writer &dst, Reader &src) {
+	vector<uint8_t> buffer(4096);
```

Comment:
I think we don't need to. In places where it matters, such as the disk writers, we will call it with an appropriately sized buffer instead of relying on this default.